server/test/user.spec.js

```diff
@@ -101,6 +102,32 @@ describe('Users Routes', () => {
       expect(res).to.be.a('object');
       expect(res.body).to.have.property('data');
     });
+
+    it('should not retrieve users functionality if authorization token is not valid', async () => {
```

Comment:
@geekdanitek Kindly give more explanation on this

Reply:
This is a test for when a user that wants to get the list of other users functionality, only logged user can make a request to the endpoint